src/Framework/Sdk/SdkResolver.cs

```diff
@@ -30,7 +30,7 @@ public abstract class SdkResolver
         ///     the SDK could not be resolved.  Return <code>null</code> if the resolver is not
         ///     applicable for a particular <see cref="SdkReference"/>.
         ///  </returns>   
-        ///  <remarks>
+        /// <remarks>
         ///    Note: You must use <see cref="Microsoft.Build.Framework.SdkResultFactory"/> to return a result.
         ///  </remarks>
```

Comment:
📝 The ending elements (`</returns>` and `</remarks>`) should be manually corrected here